src/pages/journal/components/test-outcome/test-outcome.ts

```diff
@@ -10,6 +10,17 @@ import { end2endPracticeSlotId } from '../../../../shared/mocks/test-slot-ids.mo
 import { COMMUNICATION_PAGE, OFFICE_PAGE, PASS_FINALISATION_PAGE } from '../../../page-names.constants';
 import { ModalEvent } from '../../journal-rekey-modal/journal-rekey-modal.constants';
 import { DateTime, Duration } from '../../../../shared/helpers/date-time';
+import { SlotDetail } from '../../../../shared/models/DJournal';
```

Comment:
How come we're not using the centrailsed schema from the @dvsa npm org?

Reply:
Second this - it needs to use the @dvsa npm org.  I think we should remove the Djournal building, so that code completion doesn't reference this first, and should put a stop to this going forward.

Reply:
@ammarhaiderbjss  - another thing to go on that ticket ;)

Reply:
Yep - was auto imported so have now fixed